src/pymor/models/basic.py

```diff
@@ -252,6 +289,16 @@ def __init__(self, T, initial_data, operator, rhs, mass=None, time_stepper=None,
         self.build_parameter_type(self.initial_data, self.operator, self.rhs, self.mass, provides={'_t': 0})
         self.parameter_space = parameter_space
 
+    def __str__(self):
+        return (
+            f'{self.name}\n'
+            f'    class: {self.__class__.__name__}\n'
+            f'    parameter type: {self.parameter_type}\n'
+            f'    {"" if self.linear else "non-"}linear\n'
+            f'    number of equations: {self.solution_space.dim}\n'
+            f'    number of outputs:   {len(self.outputs)}'
+        )
```

Comment:
This is quite different from `StationaryModel.__str__`. I would except both to return a very similar representation.

Reply:
You only said how you would expect `StationaryModel.__str__` to look like...

Reply:
Well, I guess I didn't have the bigger picture in mind when saying this ..

Reply:
That would be best. I could then adjust the `iosys` models.